src/pyhf/modifiers/shapefactor.py

```diff
@@ -19,6 +19,7 @@ def required_parset(cls, sample_data, modifier_data):
             'is_shared': True,
             'inits': (1.0,) * len(sample_data),
             'bounds': ((0.0, 10.0),) * len(sample_data),
+            'fixed': False,
```

Comment:
For now, systematic-level application.. but we'd want to change to `(False,)*len(sample_data)` later?